extensions-core/core/runtime/src/main/java/org/apache/camel/quarkus/core/FastCamelContext.java

```diff
@@ -53,19 +53,22 @@
 import org.apache.camel.util.ObjectHelper;
 
 public class FastCamelContext extends AbstractCamelContext implements CatalogCamelContext, ModelCamelContext {
+    private final CamelContext reference;
```

Comment:
I'd appreciate a short field level JavaDoc explaining the purpose of this field.

Reply:
Yes, I can add that, but it may be better to document the `getCamelContextReference()` method instead.

Reply:
Yes, documenting on `getCamelContextReference()` would be fine too.